draft-ietf-taps-interface.md

```diff
@@ -459,6 +459,23 @@ useful to indicate its reliability requirements on a per-Message basis.
 This property applies to Connections and Connection Groups. This is not a
 strict requirement.  The default is to not have this option.
 
+### Direction of communication
+
+Type: Enum
+
+This property specifies whether an application wants to use the connection for sending and/or receiving data.  Possible values are:
+
+Bidirectional (default):
+: The connection must support sending and receiving data
+
+unidirectional send:
+: The connection must support sending data. 
+
+unidirectional receive:
+: The connection must support receiving data
+
+In case a unidirectional connection is requested, but the system should fall back to bidirectional transport if unidirectional connections are not  supported by the transport protocol.
+
```

Comment:
Though it probably is one for reasons of implementation expediency, I’m not sure this belongs as a 

Post had these as separate connection calls, which seems more natural to me, and maps to multicast more nicely: receive-only has a lot in common with listen, for instance.